src/gui/windows/save_creation.zig

```diff
@@ -82,6 +82,7 @@ fn flawedCreateWorld() !void {
 
 		gamerules.put("default_gamemode", @tagName(gamemode));
 		gamerules.put("cheats", allowCheats);
+		gamerules.put("allow_explosives", true);
```

Comment:
Please add a checkbox for it.

Reply:
I widened the window just a bit (to 128+32) so label doesn't wrap

![image](https://github.com/user-attachments/assets/6b801832-4e4e-4b60-8fe3-771889386e82)